.github/workflows/issue_bot.yml

```diff
@@ -0,0 +1,19 @@
+name: Contribution Bot
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  auto-assign:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: 'Auto-assign issue'
+        if: github.event.issue.author_association != 'COLLABORATOR'
```

Comment:
Wenn ichs richtig sehe sind wir alle `MEMBER` und nicht `COLLABORATOR`.

Siehe Label neben unseren Kommentaren und https://docs.github.com/en/graphql/reference/enums#commentauthorassociation

Reply:
`MEMBER` wären ja alle aus der Orga. Dann würden wir nicht mitbekommen, wenn es jemand von it@m einstellt, der nicht Teil unseres Teams ist. Siehe DevTobi in https://github.com/it-at-m/digiwf-core/pull/1570

Aber der Hinweis zum Label ist gut. Meine Vermutung war, dass das "höchste" angezeigt wird. Sonst wüsste ich aktuell nicht, wie man Collaborator wird. mal googln

Reply:
Wenn ich mich direkt zum Repo hinzufüge ohne Team, dann steht dort "Added as Collaborator". Aber es steht weiterhin als Label Member da

Reply:
@simonhir Hab es angepasst

Reply:
Ich glaube Collaborator bekommt man, wenn man außerhalb der Organisation ist und Rechte bekommt.